File: Euler1/Euler1.java
import java.util.ArrayList;

public class Euler1
{
   public static void main (String []args)
   {
      int result = 0;
      for(int i = 1; i<1000; i++)
      {
         if(i%5 == 0||i%3 == 0)
         {
            result +=i;
         }
      }
      System.out.println(result);
   }
}

File: Euler1/euler1.c
#include<stdio.h>
#include<stdlib.h>

int main(void){

	int result = 0;
	for(int i = 0; i < 1000; i++){
		if(i % 3==0){
			result+=i;
		} else if(i % 5==0){
			result+=i;
		}
	}
	printf("result: %d", result);
	return 0;
}
